TweetMate/src/main/java/com/ybsystem/tweetmate/activities/MainActivity.java

```diff
@@ -81,21 +83,21 @@ public void onBackPressed() {
     public void onItemClick(int position) {
         Intent intent;
         switch (position) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a default case to this switch. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://192.168.11.19:9000/coding_rules#rule_key=squid%3ASwitchLastCaseIsDefaultCheck)